### DISCUSSION TITLE
Gogs 0.12.9 has been released!

### DISCUSSION THREAD

**Discussion by unknwon** [gogs/gogs#7026] [Announcements]
Hello everyone,

Hope you have pleasant experiences with Gogs so far, and we’re excited to announce that Gogs 0.12.9 is released!

You can find binary downloads on our [website](https://gogs.io/docs/installation/install_from_binary) and the [GitHub release](https://github.com/gogs/gogs/releases/tag/v0.12.9) page.

The CHANGELOG for this release is also available on the [GitHub release](https://github.com/gogs/gogs/releases/tag/v0.12.9) page.

Please enjoy and happy coding!

Please comment on this thread if you spot any packaging issues for the release. For general feature requests and bug reports, please use our [curated issue forms](https://github.com/gogs/gogs/issues/new/choose) :)

### Backwards compatibility

You may upgrade your installation directly to this version if you're running Gogs 0.8.5 or newer.

**Comment by MiMaUser** [comment]
Hello,
do you have any plan of next release (12.10)? I miss the functionality of the integration between YouTrack and Gogs, which was properly working in previous versions and I would like to avoid downgrade to previous one :)
Thank you

Milos

**Reply by unknwon** [reply]
0.12.10 is currently planned to be released on July 30th (not a commitment 🤫).

BTW, switching versions between 0.12.* is completely safe and compatible.
